sys-kernel/raspberrypi-image/raspberrypi-image-4.14.98_p20190215.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="1.$(ver_cut 5)"
+
+DESCRIPTION="Raspberry Pi (all versions) kernel and modules"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+SRC_URI="https://github.com/raspberrypi/firmware/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BSD"
+SLOT="0"
+KEYWORDS="-* ~arm ~arm64"
+
+RDEPEND="sys-boot/raspberrypi-firmware"
+
+S="${WORKDIR}/firmware-${MY_PV}"
+
+src_configure() {
+	:;
+}
+
+src_compile() {
+	:;
+}
+
+src_install() {
+	insinto /boot
```

Comment:
I barely believe it should be installed into boot bat rather in some /usr directory, you gonna fail if noauto for boot in fstab otherwise

Reply:
Well, in order to boot a RPi, you need a dedicated FAT32 partition, and nearly all distros are using this partition as ```/boot```, since the RPi boot loader expect there the kernel with the default names, unless changed in ```config.txt```. I am open for suggestions.  Btw, the older eBuilds were installing into ```/boot```. Maybe, we could check, if /boot is mounted?

Reply:
well, older ebuikds did it wrong way, if you wanna keep it installing straight to boot there is mount-boot eclass as well

Reply:
Thanks, that sounds good. Pushed new version.